manifests/03-clusterrole.yaml

```diff
@@ -74,6 +74,13 @@ rules:
       - namespaces
     verbs:
       - get
+  # allow the operator to retrieve alerts from alert manager
```

Comment:
can't we bind to the [`monitoring-alertmanager-edit`](https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/cluster-monitoring-operator/monitoring-alertmanager-edit-role.yaml) role instead?

Reply:
Done